test/parallel/test-timers-setunreftimeout.js

```diff
@@ -0,0 +1,61 @@
+// Flags: --expose-internals
+
+'use strict';
+
+const common = require('../common');
+
+const { strictEqual } = require('assert');
+const { setUnrefTimeout } = require('internal/timers');
+
+// ERR_INVALID_CALLBACK
+{
+  // It throws ERR_INVALID_CALLBACK when first argument is not a function.
+  common.expectsError(
+    () => setUnrefTimeout(),
+    {
+      code: 'ERR_INVALID_CALLBACK',
+      type: TypeError,
+      message: 'Callback must be a function'
+    }
+  );
+}
+
+// test case for passing the arguments
+{
+  for (let i = 0; i < 4; i++) {
+    const results = [];
+    const inputArgs = [];
+    // set the input argument params
+    for (let j = 0; j <= i; j++) {
+      inputArgs.push(j);
+    }
+
+    // For checking the arguments.length,
+    // callback function should not be arrow fucntion.
+    const timer = setUnrefTimeout(common.mustCall(
+      function(args1, args2, args3, args4) {
```

Comment:
I'd suggest to name them as `arg1`, `arg2`, etc. as `args` is usually an abbreviation for multiple arguments (like you have with `inputArgs`) and here we have a single argument.
Just a nit though.